### DISCUSSION TITLE
nas上的alist，日志出现 2023/02/22 01:12:28.369250 ERROR RESTY Post "https://api.nn.ci/ocr/file/json": tls: failed to verify certificate: x509: certificate signed by unknown authority, Attempt 4

### DISCUSSION THREAD

**Discussion by laolihb** [AlistGo/alist#3573] [General]
nas的alist，运行正常，但日志出现：

2023/02/22 01:12:28.369250 ERROR RESTY Post "https://api.nn.ci/ocr/file/json": tls: failed to verify certificate: x509: certificate signed by unknown authority, Attempt 4
需要解决吗？
如何解决？

**Comment by 14960** [comment]
我也出现X509错误

**Reply by 14960** [reply]
按照以下方法解决了：
1. 删掉Oauth令牌链接https里面的s
2. 点击添加，如果提示请求过多的错误，再次点击添加，多点几次一直点到能添加为止。
